### DISCUSSION TITLE
RFC: best practices

### DISCUSSION THREAD

**Discussion by EmanuelCampos** [mybigday/whisper.rn#90] [General]
hello, would be good to have better documentation of best practices, for example when to release context, or if just stop transcribeRealtime, production recommendations, will be happy to contribute when I understand this better

**Comment by jhen0409** [comment]
It might not be the best but I'll happy to share our production practices, I'll start the doc update on this later.

**Comment by EmanuelCampos** [comment]
Awesome @jhen0409!

could you provide me a TLDR? I'm going to production and would love to know if I'm doing it on the right way

now I'm initializing whisper when the recording starts, then stopping when the record finishes

**Comment by jhen0409** [comment]
I'm sorry for slow responses.

The [docs/TIPS.md](https://github.com/mybigday/whisper.rn/blob/main/docs/TIPS.md) will be document that will help you use whisper.rn in production. We'll keep updating it, and welcome any discussions and contributions!

Also, I'll move this issue to Discussions.

**Reply by EmanuelCampos** [reply]
Thanks for this! definitely will contribute when have I something!
